### PR TITLE
Throw exception on null template

### PR DESCRIPTION
**Pull request by shopblocks**

### CODE REVIEW

**Comment by shopblocks** [comment]
Im not sure whether there should be a test for this.

As a quick outline for what it does.

If you create a new format for tests, and then run the tests, when it gets to the 'results' section, it throws a warning saying you cannot include an  empty file (the template is null, so it tries `include null`)

This simply helps the developer find an issue, rather than trying to track it down like I did yesterday.

**Comment by davidpersson** [comment]
This is OK. Thank you for working on the patch.

(I'll fix the one minor coding standard when I'll merge this.)

**Comment by shopblocks** [comment]
Whats the code standard violation?

File: test/Report.php [hunk 1]
```diff
@@ -229,6 +230,11 @@ public function render($template, $data = array()) {
 		$template = Libraries::locate('test.templates', $template, array(
 			'filter' => false, 'type' => 'file', 'suffix' => ".{$config['format']}.php"
 		));
+
+		if($template === null) {
```
Comment (davidpersson) [inline test/Report.php]: Between the `if` and the `(` one whitespace is needed to be in line with the CS.

Comment (shopblocks) [inline test/Report.php]: Ah okay. I can fix that if you like.

Also, do you want the commits to be flattened or are you happy with the several commits being merged?

**Comment by davidpersson** [comment]
Merged in c87c87d3ea3f4616d19be17080137fdab6e2b998. Thanks for the patch.

**Pull request closed without merging**